draft-ietf-tls-tls13.md

```diff
@@ -3695,6 +3705,16 @@ max_early_data_size
   depend on being able to send large quantities of padding in early data records.
 {:br }
 
+The PSK associated with the ticket is computed as:
+
+~~~~
+    HKDF-Expand-Label(resumption_master_secret,
```

Comment:
The rule we need to maintain is that we Expand after every Extract.  Expand can be run in series safely (see the recent discussion about the exporter).